internal/cmd/cloud_login.go

```diff
@@ -309,3 +293,20 @@ func getStacks(gs *state.GlobalState, jsonRawConf json.RawMessage, token string)
 	}
 	return stacks, nil
 }
+
+func getDefaultStack(gs *state.GlobalState, jsonRawConf json.RawMessage, token string) (int64, string, error) {
+	stacks, err := getStacks(gs, jsonRawConf, token)
+	if err != nil {
+		return 0, "", fmt.Errorf("failed to get default stack slug: %w", err)
+	}
+
+	if len(stacks) == 0 {
+		return 0, "", errors.New("no stacks found for the provided token " +
+			"please create a stack in Grafana Cloud and initialize GCk6 app")
+	}
+
+	for slug, id := range stacks {
+		return int64(id), slug, nil
+	}
+	return 0, "", nil // This should never happen
```

Comment:
Probably should return an error, even if it should never happen, right?

Reply:
True! Done.